ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -161,9 +162,13 @@ export default class SendAssetRow extends Component {
           />
           <div className="send-v2__asset-dropdown__list">
             {this.renderNativeCurrency(true)}
-            {this.state.sendableTokens.map((token) =>
-              this.renderToken(token, true),
-            )}
+
+            <TokenListDisplay
```

Comment:
Just curious why did you decide to make this a new component instead of changing the `sendableTokens` to further filter out tokens without balances?